tests/e2e/test_conversion.py

```
@@ -0,0 +1,149 @@
+        "max_model_len": 64,
+        "enforce_eager": True,
+    }
+
+    run_vllm_path = str(Path(__file__).parent / "run_vllm.py")
+    vllm_env = sys.executable if VLLM_PYTHON_ENV.lower() == "same" else VLLM_PYTHON_ENV
+
+    env = os.environ.copy()
+    venv_bin = os.path.dirname(vllm_env)
+    env["PATH"] = venv_bin + os.pathsep + env.get("PATH", "")
+
+    result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)